### PR TITLE
Editor/Slides: render slide content inside svg foreignObject to display "scaled" slide

### PR DESCRIPTION
**Pull request by rwaldron**
image

### CODE REVIEW

**Comment by rwaldron** [comment]
Rebased

File: client/components/SlideComponentsList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    };
+    return (
+        <div style={style}>
+            <svg width="500" height="400">
+                {/* intentionally break out */}
+                <foreignObject
+                    transform="scale(0.5)"
+                    width="100%"
+                    height="100%"
+                >
```
Comment (ktbee) [inline client/components/SlideComponentsList/index.jsx]: This turns the HTML into an SVG everywhere `SlidesListComponents` is used, which covers places outside of the slide previews in the editor (ie in the "Preview" editor tab and on the front end view of a scenario). We'll need to limit this to the slide previews.

**Comment by ktbee** [comment]
@rwaldron really cool and impressive SVG stuff. I like that it's such a small fix.

File: client/components/SlideComponentsList/index.jsx (hunk 1 above)
Comment (rwaldron) [reply client/components/SlideComponentsList/index.jsx]: I totally missed that!

**Comment by rwaldron** [comment]
@ktbee changes made!

**Comment by ktbee** [comment]
@evmiguel want to take a look as well?

File: client/components/SlideComponentsList/index.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const SVGSlideComponentsList = ({ asSVG = false, components }) => {
+    const style = {
+        height: '150px',
+        overflow: 'hidden'
+    };
+    return asSVG ? (
+        <div style={style}>
+            <svg width="500" height="400">
+                {/* intentionally break out */}
+                <foreignObject
```
Comment (evmiguel) [inline client/components/SlideComponentsList/index.jsx]: This is cool! I've never seen the **foreignObject** tag before!

**Pull request merged**